src/basecmd.c

```diff
@@ -202,7 +211,8 @@ void
 command_get_config(uint32_t *args)
 {
     sendf("config is_config=%c crc=%u move_count=%hu is_shutdown=%c"
-          , is_finalized(), config_crc, move_count, sched_is_shutdown());
+          , is_finalized(), config_crc, move_count-reserved_moves,
+          sched_is_shutdown());
```

Comment:
Is the intent of "reserved moves" to be "stop gap" code?  If so, I think it would be preferable to do that in the host code.  (That is, have the host count how many oids need to use the move_queue and do the subtraction.)  Otherwise, it results in several iterations where users may have to reflash their micro-controllers.

Reply:
I agree that it is preferably done in host code. This is a workaround for while the `steppersync` does not recognize oids buffered in the move queue. Currently, users have to reflash the MCUs only if, e.g., soft_pwm may use the move_queue. But then they would have to re-flash anyway. The actual number of "reserved" slots in the move queue is determined on how many `move_queue`s there are. 

I didn't want to touch the host code because all changes there would be obsolete once the `steppersync` upgrade comes. But if you say it was better for cleanlyness between PRs, I can try implementing a minimalistic host-side count.